### DISCUSSION TITLE
How much effectively in dollar terms is an Oscar worth if a recipient tries to sell it?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#384] [Galaxy Trivia] [answered]
**Hint:** to Academy

**Comment by peterbaccaro** [comment, marked as answer]
Just **$1**, this is because the official regulations state that winners cannot sell their Oscar without first offering to sell it back to the Academy for **$1**.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [x3 Galaxy Brain 🥈](https://github.com/peterbaccaro?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Gold for 32 answers), and don't hesitate to post your own questions to the community. 😊🎉
